apps/server/src/modules/common-cartridge/service/common-cartridge-export.service.ts

```diff
@@ -1,9 +1,13 @@
 import { FileDto, FilesStorageClientAdapterService } from '@modules/files-storage-client';
 import { Injectable } from '@nestjs/common';
+import { BoardClientAdapter } from '../common-cartridge-client/board-client/board-client.adapter';

```

Comment:
Why no import from the index file?

Reply:
VS Code auto import I missed. I changed it to use the index file.